### ISSUE TITLE
Needs to be updated for the new .Net 8 templates

### ISSUE THREAD

**Issue by kevon-vems** [dotnet/AspNetCore.Docs#31271]
### Description

Current documentation still refers to cshtml documents, but the new templates are all using razor files.

Alternately, if this article is not for Blazor, then the template which has the links to the KB articles needs to be updated to point at the correct one.

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/security/authentication/accconfirm?view=aspnetcore-8.0&tabs=visual-studio

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authentication/accconfirm.md

### Document ID

98e7aacb-2a01-d581-9b03-bfef2893163e

### Article author

@Rick-Anderson

**Comment by guardrex** [comment]
cc: @JeremyLikness @halter73 ... Is there a plan to address this for Blazor Web Apps? We can place coverage in the Blazor *Security* node if not (and I'll update the metadata here if that's the case). If we do that, then the KB link that @kevon-vems is referring to can receive an updated link.

@kevon-vems ... Can you be more specific on "the template which has the links to the KB articles"? Do you mean the Blazor Web App template?

I see one thing that requires a quick update. The scaffolding guidance still claims that there aren't Razor components for Identity UI. I'll take care of that update now.

**Comment by kevon-vems** [comment]
@JeremyLikness Yes, the blazor web app template has links to what appears to be the MVC related KB.  

I'm working on creating a project with Identity and there are several links in the Blazor Web App template that seem not to be quite right

**Comment by guardrex** [comment]
I'm not surprised. We haven't reached it yet. I've been putting out 🔥 ever since release, and this is another one that we'll need to address *ASAP* 🚓🚒🚑. It won't be until after the 🎆 holidays tho. Everyone has just about bugged out until '24, and I'll be doing the same shortly.

@Rick-Anderson ... I'll take this one, and I suspect that we'll ...

* Create new coverage in the Blazor *Security and Identity* node.
* Have the template links updated to point to the new coverage.

**Comment by guardrex** [comment]
**UPDATE (1/2)**: I'm back from vacation, and I'll get to this as soon as I can. 🏃🏃🏃🏃

**Comment by kevon-vems** [comment]
Awesome!!  Hope you had a great holiday!

I was wondering if you could point me at the code for the templates. I was considering altering them and creating a PR for a different issue I have.

**Comment by guardrex** [comment]
If you mean the framework's project templates, they're here ...

https://github.com/dotnet/aspnetcore/tree/main/src/ProjectTemplates

**Comment by guardrex** [comment]
I've looked things over, and I think we'll update ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/security/server/?view=aspnetcore-8.0&tabs=visual-studio#blazor-identity-ui-individual-accounts

... with a cross-link to the components in the framework repo, specifically the `Account` folder, which is above the components in the `Pages` and `Shared` folders ...

https://github.com/dotnet/aspnetcore/tree/main/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Account

Devs will be able to quickly reach the code to see what these Identity components look like without having to actually scaffold them into an app for inspection.

The naming makes the purpose of each component self-exclamatory. Implementations are subject to churn in the immediate future, so detailed explanations probably aren't wise at this point.

@kevon-vems, you didn't say ***exactly*** what URL those links are or where they are in the project template. I didn't see them in a casual inspection. If you know the exact link(s), could you state that here? I can then search their repo for them, identify the ones that are in the BWA template, and ask them update them.
